src/main/archetype/ui.tests/test-module/lib/wdio.commands.js

```diff
@@ -133,6 +133,12 @@ browser.addCommand('AEMSitesSetPageTitle', function(parentPath, name, title) {
     return originalTitle;
 });
 
+browser.addCommand('focus', function () {
```

Comment:
I don't think you require a command for this, you could do it inline where required. Command should be used for some high level user functionality, like aemLogin, deleteAsset etc

Reply:
Done